pygls/types.py

```diff
@@ -1236,6 +1251,15 @@ def __init__(self,
         self.context = context
 
 
+class HoverParams(TextDocumentPositionParams):
+    def __init__(self,
+                 textDocument: 'TextDocumentIdentifier',
```

Comment:
this parameter should be named `text_document`, for consistency with the rest of this module.